setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
+extend-ignore = E203
```

Comment:
Is Flake used or supposed to be used? If yes, it could be mentioned in the `README` or some other file, e.g. `CONTRIBUTING`.

Reply:
I think that was leftovers from somewhere. The usual black, isort and flake could be added in the future.